### PR TITLE
XP-257 cleanup cproto dir

### PR DESCRIPTION
**Pull request by wilk10**
### References

[XP-257](https://xainag.atlassian.net/browse/XP-257)

### Summary

Making sure only autogenerated *_pb2.py* files are in `cproto` module

### Are there any open tasks/blockers for the ticket?

None

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [x] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [x] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the merge request title or the references section.
- [x] Added an informative merge request summary.

**Code checklist**

- [x] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.
- [x] Conforms to Google docstring style.
- [x] Conforms to XAIN structlog style.

### CODE REVIEW

File: xain_fl/coordinator/legacy_participant.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 """Module implementing the networked Participant using gRPC.
 """
 
-# TODO: Currently, the participant is completely ignored in the mypy.ini, since the participant
-# will be removed anyways in XP-208.
+# TODO: this is an outdated version of the Participant state machine.
```
Comment (atymoshchuk) [inline xain_fl/coordinator/legacy_participant.py]: Do we need to keep legacy code?

Comment (tanertopal) [reply xain_fl/coordinator/legacy_participant.py]: This: https://github.com/xainag/xain-sdk/pull/13 contains the most current code changes
This: https://github.com/xainag/xain-sdk/pull/21 will delete it from xain-sdk

So it need to be moved here. So this file or another needs to contain the most current version. Ergo it was outdated but is now soon to be replaced

Comment (wilk10) [reply xain_fl/coordinator/legacy_participant.py]: i'll solve it as part of https://xainag.atlassian.net/browse/XP-373 let's merge this PR first

**Comment by wilk10** [comment]
XP-257 has been merged to development already. closing this PR

**Pull request closed without merging**